flux/index.js

```diff
@@ -3,11 +3,13 @@ import createLogger from 'redux-logger'
 import reduxPromise from 'redux-promise'
 import reduxThunk from 'redux-thunk'
 import {AsyncStorage} from 'react-native'
+import {allViews} from '../views/views'
 
 export const SAVE_HOMESCREEN_ORDER = 'SAVE_HOMESCREEN_ORDER'
+let defaultViewOrder = allViews.map(v > v.view)
```

Comment:
What are you asking here? I haven't had a chance to run anything, so I went with your changes listed above.